css/style.css

```diff
@@ -220,14 +225,18 @@ main {
   border-radius: 2px;
 }
 
-.toggle-inner {
+input[type="checkbox"] {
+  display: none;
```

Comment:
> Doesn't this mean you still get the accessibility benefits?

No tabbing and space to select won't work.

> Also importantly: there will be no space allocated for it between the other elements.

I'm not sure what you mean, I assume you mean it will be bad in the layout. Firefox just fixed allowing `-moz-appearance: none; height: 0;..` I think, if not there are other ways like absolute positioning (valid use case if other methods of removing from the layout doesn't work).

>  prevent the checkbox from taking up physical space on the page

Well `margin` `border` `padding` should likely be applied to all elements anyway.
A few attributes aren't really an issue, if we need the same styles again we can always create another class like `.visually-hidden`.

`visibility: hidden;` also makes the checkbox non interactive too so that is out.

> the element still has a border around it; even though inspect tells me that the border:0 rule is being enforced.

Yeah check `appearance: none;` if not hide it off page with indent or positioning. Form fields like a few other elements have special appearance by default which makes these styles persist. To my knowledge they literally just fixed the height and width resetting too which should hopefully help.